Source/Core/DolphinQt/GameList/GridProxyModel.h

```diff
@@ -15,4 +15,5 @@ class GridProxyModel final : public QSortFilterProxyModel
   explicit GridProxyModel(QObject* parent = nullptr);
   QVariant data(const QModelIndex& index, int role = Qt::DisplayRole) const override;
   bool filterAcceptsRow(int source_row, const QModelIndex& source_parent) const override;
+  bool lessThan(const QModelIndex& left, const QModelIndex& right) const;
```

Comment:
```suggestion
  bool lessThan(const QModelIndex& left, const QModelIndex& right) const override;
```
Eliminates a warning.